reference/5.1/Microsoft.PowerShell.Core/About/about_PowerShell_exe.md

```diff
@@ -170,17 +166,49 @@ values, see [about_Execution_Policies](about_Execution_Policies.md).
 
 #### -Command
 
-Executes the specified commands (with any parameters) as though they were typed
-at the PowerShell command prompt. After execution, PowerShell exits unless the
-**NoExit** parameter is specified. Any text after `-Command` is sent as a
-single command line to PowerShell. This is different from how `-File` handles
-parameters sent to a script.
+Executes the specified commands (and any parameters) as though they were typed
+at the PowerShell command prompt, and then exits, unless the `NoExit`
+parameter is specified.
+
+The value of **Command** can be `-`, a script block, or a string. If the value
+of **Command** is `-`, the command text is read from standard input.
+
+The **Command** parameter only accepts a script block for execution when it can
+recognize the value passed to **Command** as a **ScriptBlock** type. This is
+_only_ possible when running `pwsh` from another PowerShell host. The
+**ScriptBlock** type may be contained in an existing variable, returned from an
+expression, or parsed by the PowerShell host as a literal script block enclosed
+in curly braces (`{}`), before being passed to `powershell.exe`.
 
-The value of **Command** can be "-", a string, or a script block. The results
-of the command are returned to the parent shell as deserialized XML objects,
-not live objects.
+```powershell
+powershell -Command {Get-WinEvent -LogName security}
+```
 
-If the value of **Command** is "-", the command text is read from standard
+In `cmd.exe`, there is no such thing as a script block (or **ScriptBlock**
+type), so the value passed to **Command** will _always_ be a string. You can
+write a script block inside the string, but instead of being executed it will
+behave exactly as though you typed it at a typical PowerShell prompt, printing
+the contents of the script block back out to you.
+
+A string passed to **Command** is still executed as PowerShell code, so the
+script block curly braces are often not required in the first place when
+running from `cmd.exe`. To execute an inline script block defined inside a
+string, the [call operator](about_operators.md#special-operators) `&` can be
+used:
+
+```
```

Comment:
Should this be a powershell codefence?

Reply:
This can be any shell but the preceding paragraphs are talking about cmd.exe.